apps/lcs/translations/src/en/fields.json

```diff
@@ -14,6 +14,17 @@
     "legend": "When did the tenant move into your property?",
     "hint": "For example 21 3 2015"
   },
+  "before-or-after-1988": {
+      "legend": "Has the prospective tenant been in the UK since before 1988?",
```

Comment:
text copy is different in design files, please double check the correct version.  Should be "Has the tenant been..."

Reply:
Use `isPageHeading: true` for provide Heading and Title for the page

Reply:
missing class `govuk-radios govuk-radios--inline` for horizontal alignment as per design.